apps/bot-discord/src/index.js

```diff
@@ -80,3 +80,12 @@ for (const file of helperEventFiles) {
 }
 
 client.login(process.env.DISCORD_TOKEN);
+
+export function checkForPerms(roles) {
```

Comment:
Why is the function over here? It should be somewhere else like `utils/checkSupporter.js`, not in the index file.

An another complaint is that you're not using the `for (const ... in ...)` syntax.

Lastly, you should use `<RoleManager>.cache.get` instead of turning it into JSON.